src/api/searchCities.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import SearchCityTypes from '../utils/constans';
+
+interface ItemsCity {
+    id: string;
+    language: string;
+    localityType: string;
+    resultType: string;
+    title: string;
+    highlights: object;
+    address: {
+        city: string;
+        countryCode: string;
+        countryName: string;
+        county: string;
+        label: string;
+        state: string;
+    };
+}
+
+async function getSearch(search: string) {
+    const path = `${process.env.REACT_APP_HERE_API_BASE_URL}autocomplete?q=${search}&apiKey=${process.env.REACT_APP_API_KEY}&types=${SearchCityTypes.CITY}&in=circle:${SearchCityTypes.CIRCLE};r=${SearchCityTypes.RADIUS}&in=countryCode:${SearchCityTypes.COUNTRY_CODE}&limit=${SearchCityTypes.LIMIT}&lang=${SearchCityTypes.LANG}`;
```

Comment:
this string is too long. check an example of get query here https://github.com/axios/axios#note-commonjs-usage